packages/canary-web-components/src/components/ic-pagination-bar/ic-pagination-bar.tsx

```diff
@@ -71,7 +71,7 @@ export class PaginationBar {
   /**
    * The label which will be used in place of 'items' if type is data. Should be capitalised.
    */
-  @Prop() itemLabel?: string = "Item";
+  @Prop() rangeItemLabel?: string = "Item";
```

Comment:
I keep on reading this prop in my head and wondering if it should be `range-label-item / rangeLabelItem`? Will let you decide...